Katello/katello
# Fixes #36629 - Add cvv synced to env created at on capsule sync page

#### What are the changes introduced in this pull request?

* The published date should be for the version that's tied to the env in that capsule
* Before it was the last published date of the latest version of the CV

#### Considerations taken when implementing this change?

N/A

#### What are the testing steps for this pull request?

* Check out PR
* Create a CV, publish and promote it to an env associated with a smart-proxy so it syncs
* Go to Infrastructure --> Smart Proxies --> <your smartproxy) --> Content --> Expand environment
* Check if the created at matches the cv version synced to the env instead of the initial cv created at time
